CreeDictionary/CreeDictionary/paradigm/panes.py

```diff
@@ -139,6 +144,17 @@ def fill(self, forms: dict[str, Collection[str]]) -> Paradigm:
         """
         return Paradigm(pane.fill(forms) for pane in self.panes)
 
+    def as_static_paradigm(self) -> Paradigm:
+        f"""
```

Comment:
Is the `f` intentional?
```suggestion
        """
```

Reply:
Nope 🙃